clippy_lints/src/map_err_ignore.rs

```diff
@@ -135,7 +135,7 @@ impl<'tcx> LateLintPass<'tcx> for MapErrIgnore {
                                     body_span,
                                     "`map_err(|_|...` ignores the original error",
                                     None,
-                                    "Consider wrapping the error in an enum variant",
+                                    "Consider wrapping the error in an enum variant for more error context, or using a named wildcard (`.map_err(|_ignored| ...`) to intentionally ignore the error",
```

Comment:
```suggestion
                                    "Consider storing the original error as a source in new error, or silence this warning using a named wildcard (`.map_err(|_ignored| ...`)",
```

Reply:
I like that! My one concern is (and I wasn't sure what the best term was) "named wildcard" is that most correct/easily identifiable here?

Reply:
I think a more accurate term might be "ignored identifier" instead of "wildcard pattern"

Reply:
there i updated the suggestions